### PR TITLE
Build all images from alpine image

### PR DESCRIPTION
**Pull request by at-wat**

### CODE REVIEW

**Comment by at-wat** [comment]
@ykoga-kyutech please take a look

File: kinetic/ros-core/Dockerfile [hunk 1]
```diff
@@ -1,3 +1,29 @@
-FROM alpineros/alpine-ros:kinetic-bare
```
Comment (ykoga-kyutech) [inline kinetic/ros-core/Dockerfile:1]: Just a question, is there any reason why you don't use `alpineros/alpine-ros:kinetic-bare` any more?

**Review by ykoga-kyutech — APPROVED** [review]
@at-wat Code itself looks good to me. Added question.

File: kinetic/ros-core/Dockerfile (hunk 1 above)
Comment (at-wat) [reply kinetic/ros-core/Dockerfile:1]: On docker hub, such kind of dependency is not automatically solved. Manual build trigger is required to build all images on the previous configuration.

Comment (ykoga-kyutech) [reply kinetic/ros-core/Dockerfile:1]: Got it. Thanks.

**Pull request merged**